hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerTask.java

```diff
@@ -118,7 +121,7 @@ public class ContainerBalancerTask implements Runnable {
   private int nextIterationIndex;
   private boolean delayStart;
   private List<ContainerBalancerTaskIterationStatusInfo> iterationsStatistic;
-
+  private OffsetDateTime currentIterationStarted;
```

Comment:
It looks like other fields in this class. I think only one thread will get access to this field because it is located in Runnable.
<img width="720" alt="image" src="https://github.com/user-attachments/assets/ed203b1f-28dc-4b5f-a746-8a0b2a2b4bdb">